support-frontend/assets/helpers/theGrid.js

```diff
@@ -107,6 +107,7 @@ export const imageCatalogue: {
   giftingGlobe: '4a7ee6f430a29fc8ac8d9f0667cfafc67b6aba46/145_148_1203_1201',
   printFeaturePackshot: '017a2f5c27394635b53c414962bbb775ce9b131d/5_39_1572_861',
   printCampaign2020: 'e50943f03b58caf6fde78d81eb65c92c048701c9/1844_0_1674_1006',
+  printCampaignHero: 'fff86e98dbe83b36892baf7942f30fd3bcbcaee6/286_215_714_285',
```

Comment:
Are there any images we can delete now?

Reply:
Probably - slightly scared to... 😬

Reply:
Be bold!

Reply:
I have added a [trello card](https://trello.com/c/he4gD84x/3150-remove-unused-items-from-thegridjs-imagecatalogue-object) to do this separately as it may need some careful checking and there is a bit of time pressure on this piece of work.